overview/destenation-page/destenation.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" type="image/png" href="img/favicon.png" />
+
+    <title>Natours | Exciting tours for adventurous people</title>
+  </head>
+  <body>
+    <main class="main">
+      <div class="card-container">
+        <div class="card">
+          <div class="card__header">
+            <div class="card__picture">
+              <div class="card__picture-overlay">&nbsp;</div>
+              <img
+                src="img/tours/tour-4-1.jpg"
+                alt="Tour 1"
+                class="card__picture-img"
+              />
```

Comment:
the image there is not shown, 

```suggestion
              <img
                src="./img/tours/tour-1-cover.jpg"
                alt="Tour 1"
                class="card__picture-img"
              />
```

Reply:
Done